README.md

```diff
@@ -187,51 +187,60 @@ const AppRoot = () => {
 
 Now, whenever you call `fetch`, the cookie will automatically be sent with the request. Descope backend SDKs also support extracting the token from the `DS` cookie.
 
-## Run a local example
+## Code Example
 
-There is a simple app that uses Descope React SDK, with two routes
+You can find an example react app in the [examples folder](https://github.com/descope/react-sdk/blob/main/examples).
```

Comment:
I think you can use relative path here
e.g. `./examples`

Reply:
cool, will change in a dedicate PR with more enhancements for readme